### PR TITLE
Added a note on supported Python version, fixes #3784

### LINKED ISSUE #3784: add a note on supported Python version to docs
-- probably under Miscellany
-- related to #3749 
-- Python 2.7 atm

### PR DESCRIPTION
**Pull request by pondrejk**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -229,6 +229,8 @@ Ready to start reviewing pull requests? We have :doc:`a guide </reviewing_PRs>`
 for that too! Finally, the :doc:`API reference </api/index>` covers individual
 functions, classes, methods and modules.
 
+**Robottelo** is compatible with Python 2.7.
```
Comment (rochacbruno) [inline docs/index.rst]: @rplevka @JacobCallahan @lpramuk I remember some of you talking about using Python 3 on testing? should we include Python 3.5 as compatible?

Comment (sghai) [inline docs/index.rst]: I'm running robottelo with python 2.7.5. Any one tried with python3.5 ? If so and didn't see any issue with that then please update that too.

Comment (omaciel) [inline docs/index.rst]: First thing, we can install `Robottelo` and dependencies on `3.5`:

```console
(robottelo) hacking/robottelo [master] » python --version
Python 3.5.1
(robottelo) hacking/robottelo [master] » pip install -r requirements.txt
.
.
.
Installing collected packages: fauxfactory, inflection, six, pyparsing, packaging, pyxdg, requests, blinker-herald, nailgun, cachetools, pyasn1, idna, pycparser, cffi, cryptography, inflector, blinker, numpy, paramiko, pygal, py, pytest, python-bugzilla, PyYAML, selenium, linecache2, traceback2, argparse, unittest2, robottelo
  Running setup.py install for nailgun ... done
  Running setup.py develop for robottelo
Successfully installed PyYAML-3.12 argparse-1.4.0 blinker-1.4 blinker-herald-0.2.0 cachetools-1.1.6 cffi-1.8.2 cryptography-1.5 fauxfactory-2.0.9 idna-2.1 inflection-0.3.1 inflector-2.0.11 linecache2-1.0.0 nailgun-0.27.0 numpy-1.11.1 packaging-16.7 paramiko-2.0.2 py-1.4.31 pyasn1-0.1.9 pycparser-2.14 pygal-2.3.0 pyparsing-2.1.8 pytest-3.0.2 python-bugzilla-1.2.2 pyxdg-0.25 requests-2.11.1 robottelo selenium-2.48.0 six-1.10.0 traceback2-1.4.0 unittest2-1.1.0

(robottelo) hacking/robottelo [master] » pip install -r requirements-optional.txt
.
.
.
Successfully built sauceclient termcolor websocket-client wrapt lazy-object-proxy MarkupSafe
Installing collected packages: coverage, docopt, coveralls, pyflakes, mccabe, pycodestyle, flake8, pbr, mock, pytest-cov, apipkg, execnet, pytest-xdist, termcolor, Click, testimony, virtualenv, pluggy, tox, isort, wrapt, lazy-object-proxy, astroid, pylint, pytz, babel, imagesize, MarkupSafe, Jinja2, Pygments, docutils, snowballstemmer, alabaster, sphinx, websocket-client, docker-py, sauceclient, import-string, manage
Successfully installed Click-6.6 Jinja2-2.8 MarkupSafe-0.23 Pygments-2.1.3 alabaster-0.7.9 apipkg-1.4 astroid-1.4.8 babel-2.3.4 coverage-4.2 coveralls-1.1 docker-py-1.9.0 docopt-0.6.2 docutils-0.12 execnet-1.4.1 flake8-3.0.4 imagesize-0.7.1 import-string-0.1.0 isort-4.2.5 lazy-object-proxy-1.2.2 manage-0.1.12 mccabe-0.5.2 mock-2.0.0 pbr-1.10.0 pluggy-0.3.1 pycodestyle-2.0.0 pyflakes-1.2.3 pylint-1.6.4 pytest-cov-2.3.1 pytest-xdist-1.15.0 pytz-2016.6.1 sauceclient-0.2.1 snowballstemmer-1.2.1 sphinx-1.4.6 termcolor-1.1.0 testimony-1.2.0 tox-2.3.1 virtualenv-15.0.3 websocket-client-0.37.0 wrapt-1.10.8
```

**Comment by omaciel** [comment]
Basic UI test:

```console
(robottelo) hacking/robottelo [master●] » py.test tests/foreman/ui/test_login.py
===================================================== test session starts ======================================================
platform darwin -- Python 3.5.1, pytest-3.0.2, py-1.4.31, pluggy-0.3.1
rootdir: /Users/omaciel/hacking/robottelo, inifile:
plugins: cov-2.3.1, xdist-1.15.0
collected 2 items

tests/foreman/ui/test_login.py ..

================================================== 2 passed in 24.65 seconds ===================================================

(robottelo) hacking/robottelo [master●] » py.test tests/foreman/ui/test_architecture.py -k test_positive_create_with_os
===================================================== test session starts ======================================================
platform darwin -- Python 3.5.1, pytest-3.0.2, py-1.4.31, pluggy-0.3.1
rootdir: /Users/omaciel/hacking/robottelo, inifile:
plugins: cov-2.3.1, xdist-1.15.0
collected 6 items

tests/foreman/ui/test_architecture.py F

=========================================================== FAILURES ===========================================================
______________________________________ ArchitectureTestCase.test_positive_create_with_os _______________________________________

self = <tests.foreman.ui.test_architecture.ArchitectureTestCase testMethod=test_positive_create_with_os>

    @run_only_on('sat')
    @tier1
    def test_positive_create_with_os(self):
        """Create a new Architecture with OS

            @id: 6c386230-2285-4f41-a3a5-6a17ae844f80

            @Assert: Architecture is created
            """
        with Session(self.browser) as session:
            for test_data in valid_arch_os_names():
                with self.subTest(test_data):
                    entities.OperatingSystem(
                        name=test_data['os_name']).create()
                    make_arch(session, name=test_data['name'],
>                             os_names=[test_data['os_name']])

tests/foreman/ui/test_architecture.py:62:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
robottelo/ui/factory.py:521: in make_arch
    Architecture(session.browser).create(**create_args)
robottelo/ui/architecture.py:27: in create
    self.configure_entity(os_names, FILTER['arch_os'])
robottelo/ui/base.py:243: in configure_entity
    filter_key, entity_locator, entity_list)
robottelo/ui/base.py:224: in select_deselect_entity
    self.click((strategy, value % entity))
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <robottelo.ui.architecture.Architecture object at 0x10ba4c7f0>
target = ('xpath', "//label[normalize-space(.)='qhspLAkMUq']/input[@type='checkbox']"), wait_for_ajax = True, ajax_timeout = 30
waiter_timeout = 12, scroll = True

    def click(self, target, wait_for_ajax=True,
              ajax_timeout=30, waiter_timeout=12, scroll=True):
        """Locate the element described by the ``target`` and click on it.

            :param tuple || WebElement target: Could be either locator that
                describes the element or element itself.
            :param wait_for_ajax: Flag that indicates if should wait for AJAX after
                clicking on the element
            :param ajax_timeout: The amount of time that wait_fox_ajax should wait.
                This will have effect if ``wait_fox_ajax`` parameter is ``True``.
            :param waiter_timeout: The amount of time that wait_until_element
                should wait. That value should be specified when non-default delay
                is needed (e.g. long run procedures)
            :param scroll: Decide whether scroll to element in case it is located
                out of the page
            :raise: UINoSuchElementError if the element could not be found.

            """
        if isinstance(target, (tuple, Locator)):
            element = self.wait_until_element(target, timeout=waiter_timeout)
        else:
            element = target
        if element is None:
            raise UINoSuchElementError(
                u'{0}: element was not found while trying to click'
>               .format(type(self).__name__)
            )
E           robottelo.ui.base.UINoSuchElementError: Architecture: element was not found while trying to click

robottelo/ui/base.py:614: UINoSuchElementError
```

**Comment by JacobCallahan** [comment]
time to get our async functions ready!
```async def test_positive_create_with_os```

**Comment by sghai** [comment]
@omaciel : ah thank you for quick test.
Okay, Let's stick to python 2.7 as supported version for now. 

ACK for this PR.

**Comment by renzon** [comment]
ACK

File: docs/index.rst (hunk 1 above)
Comment (elyezer) [inline docs/index.rst]: I use robottelo only with Python 3.4 (currently installed on my machine) it works fine with both Python 2 and 3. Also it is checked on travis for both versions.

**Pull request merged**